repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/TestConfiguration.java

```diff
@@ -1,36 +1,11 @@
 package uk.gov.hmcts.ccd.definition.store.repository;
 
-import com.opentable.db.postgres.embedded.EmbeddedPostgres;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import javax.sql.DataSource;
-import java.io.IOException;
-import java.util.concurrent.ThreadLocalRandom;
-
 @Configuration
 public class TestConfiguration {
 
-    private static final Logger LOG = LoggerFactory.getLogger(TestConfiguration.class);
-
-    @Bean
-    public DataSource dataSource() throws IOException {
-
-        int port = randomPort();
-
-        LOG.info("Starting postgres with port number {}", port);
-
-        final EmbeddedPostgres pg = EmbeddedPostgres
-            .builder()
-            .setPort(randomPort())
-            .start();
-
-        return pg.getPostgresDatabase();
-    }
+    public void emptyConfig() {
```

Comment:
This class can be deleted but that would add another 14 test files into this PR change list. Will do that refactoring in the next PR :)